cloudnetpy/instruments/rpg.py

```diff
@@ -35,13 +37,13 @@ def rpg2nc(path_to_l1_files, output_file, site_meta):
     rpg = Rpg(one_day_of_data, site_meta)
     rpg.linear_to_db(('Ze', 'antenna_gain'))
     output.update_attributes(rpg.data, RPG_ATTRIBUTES)
-    _save_rpg(rpg, output_file)
+    _save_rpg(rpg, output_file, keep_uuid)
 
 
 def get_rpg_files(path_to_l1_files):
     """Returns list of RPG Level 1 files for one day - sorted by filename."""
     files = os.listdir(path_to_l1_files)
-    l1_files = [f"{path_to_l1_files}{file}" for file in files if file.endswith('LV1')]
+    l1_files = ['/'.join((path_to_l1_files, file)) for file in files if file.endswith('LV1')]
```

Comment:
Change is ok, but was it necessary. Was line 44 missing '/'?

Reply:
I have started to prefer `'/'.join(()) ` instead of simple concatenation. It is safer method. With `join` you don't have to verify that the path coming in ends with a slash.